### PR TITLE
Update to Python 3.10

### PR DESCRIPTION
**Pull request by kariharju**
- python=3.10.12
- pip=23.2.1
- truststore=0.7.0
- rpaframework==27.1.0
- rpaframework-assistant==3.0.0
- openai==0.28.0

~~TODO: check the robot (add test run to Control Room)~~

Robot was tested and runs succesfully

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Robocorp Technologies, Inc.
+   Copyright [yyyy] [name of copyright owner]
```
Comment (bogdancondurache) [inline LICENSE:189]: is this change on purpose?

**Review by bogdancondurache — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (kariharju) [reply LICENSE:189]: Yes, if the users use template as a base without looking at the file it is a bit weird if the license automatically states Robocorp
